scio-smb/src/test/java/org/apache/beam/sdk/extensions/smb/AvroBucketMetadataTest.java

```diff
@@ -51,13 +51,16 @@
           false,
           Lists.newArrayList(
               new Schema.Field("countryId", Schema.create(Type.BYTES), "", ""),
-              new Schema.Field("postalCode", Schema.createUnion(Schema.create(Type.NULL), Schema.create(Type.BYTES)), "", ""),
+              new Schema.Field("postalCode", Schema
+                  .createUnion(Schema.create(Type.NULL), Schema.create(Type.BYTES)), "", ""),
               new Schema.Field(
                   "prevCountries",
                   Schema.createArray(Schema.create(Schema.Type.STRING)),
                   "",
                   Collections.<String>emptyList())));
 
+  private static final Schema LOCATION_UNION_SCHEMA = Schema.createUnion(Schema.create(Type.NULL), LOCATION_SCHEMA);
```

Comment:
we want to test the two conditions, when the non-leaf field is a union type and when it is not